src/server.ts

```diff
@@ -0,0 +1,156 @@
+import {db} from "./db-module";
+import {processDataToFormat} from "./data-processor";
+const request             = require('request-promise');
+const R                   = require('ramda');
+
+const USERS_TABLE_NAME: string = "github_users";
+const LIKED_LANGUAGES_NAME: string = "liked_languages";
+
+//Interface for the user objects
+interface GithubUsers{ 
+  id : number,
+  login: string,
+  name: string,
+  location: string,
+  company: string,
+  followers: number
+};
+
+/**Creates the table with name USERS_TABLE_NAME to store users*/
+const createUsersTable = () => {
+  db.none(`CREATE TABLE ${USERS_TABLE_NAME} (id BIGSERIAL PRIMARY KEY, login TEXT UNIQUE, name TEXT, location TEXT,company TEXT, followers INT)`)
+  .catch((e: Error) => {
+    errorHandler(e);
+  })
+  .then(() => process.exit(0))
+}
+
+/**Creates the table with name LIKED_LANGUAGES_NAME to store
+ * the relationship between users and languages they like
+ */
+const createLikedLanguagesTable = () => {
+  db.none(`CREATE TABLE ${LIKED_LANGUAGES_NAME} (id SERIAL PRIMARY KEY, user_id INT NOT NULL, language TEXT NOT NULL, UNIQUE (user_id, language),
+          CONSTRAINT fk_user FOREIGN KEY(user_id) REFERENCES ${USERS_TABLE_NAME}(id))`)
+    .catch((e: Error) => {
+      errorHandler(e);
+    })
+    .then(() => process.exit(0))
+}
+
+/**
+ * Adds a new entry in LIKED_LANGUAGES_NAME to represent a <language>
+ * that the <user> likes
+ * @param user Login of the user
+ * @param language language name
+ */
+const addLikedLanguage = (user: string, language: string) => {
+  db.one(
```

Comment:
What happens if the user provides a malicious user string?

Reply:
The user could atack the database with using sql injection.
I have changed the code to use preprared statements from the pg-promise library on every query that depends on user input.
Will commit the changes together with the other ones